clients/src/main/resources/common/message/LeaveGroupResponse.json

```diff
@@ -19,11 +19,21 @@
   "name": "LeaveGroupResponse",
   // Version 1 adds the throttle time.
   // Starting in version 2, on quota violation, brokers send out responses before throttling.
-  "validVersions": "0-2",
+  // Starting in version 3, we will make leave group request into batch mode.
+  "validVersions": "0-3",
   "fields": [
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "1+", "ignorable": true,
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
-    { "name": "ErrorCode", "type": "int16", "versions": "0+",
-      "about": "The error code, or 0 if there was no error." }
+    { "name": "ErrorCode", "type": "int16", "versions": "0-2",
+      "about": "The error code, or 0 if there was no error." },
+    { "name": "MemberResponseListV0", "type": "[]MemberResponse", "versions": "3",
+      "about": "List of leaving member responses.", "fields": [
+      { "name": "MemberId", "type": "string", "versions": "3",
+        "about": "The member ID to remove from the group." },
+      { "name": "GroupInstanceId", "type": "string", "versions": "3", "nullableVersions": "3",
+        "about": "The group instance ID to remove from the group." },
+      { "name": "ErrorCode", "type": "int16", "versions": "3",
```

Comment:
It might be nice to document somewhere which errors are possible and at which level. For example, NOT_COORDINATOR errors would be at the top level?

Reply:
I will document the error codes, but I don't think we need to document which error goes which level, because we are fully relying on `members` to return batch result for all members even when there are duplicate.

Reply:
Documented in the `LeaveGroupRequest` class instead.